thirdeye/thirdeye-frontend/app/pods/components/cube-tooltip/template.hbs

```diff
@@ -0,0 +1,21 @@
+<div class="te-tooltip__body te-tooltip__body--large">
+  <div>
+    <span class="te-tooltip__title te-tooltip__title--small">Definition of Terms</span>
+  </div>
+  <p>CV: current value<br/>
+    PV: previous (baseline) value<br/>
+    SUM_CV: total value<br/>
+    SUM_PV: total previous (baseline) value
+  </p>
+  <div>
+    <span class="te-tooltip__title te-tooltip__title--small">Formulas</span>
+  </div>
+  <p>Percentage Change: (CV -PV) * 100 / PV<br/>
+    Change in Contribution: (CV/SUM_CV -PV/ SUM_PV) * 100<br/>
+    Contribution to Overall Change: (CV -PV) * 100 / (SUM_CV-SUM_PV)
```

Comment:
Can you ensure there is a space before & after all the operators?

Reply:
Same comment here :-)